<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="117889" started-at="2020-06-28T04:29:00Z" finished-at="2020-06-28T04:30:58Z">
    <groups>
    </groups>
    <test name="Validating navigation Bar" duration-ms="102034" started-at="2020-06-28T04:29:00Z" finished-at="2020-06-28T04:30:42Z">
      <class name="Academy.validateNavigationBar">
        <test-method status="PASS" signature="initialize()[pri:0, instance:Academy.validateNavigationBar@5fbe4146]" name="initialize" is-config="true" duration-ms="89109" started-at="2020-06-28T09:59:00Z" finished-at="2020-06-28T10:00:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method status="PASS" signature="validateAppNavBar()[pri:0, instance:Academy.validateNavigationBar@5fbe4146]" name="validateAppNavBar" duration-ms="530" started-at="2020-06-28T10:00:29Z" finished-at="2020-06-28T10:00:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateAppNavBar -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:Academy.validateNavigationBar@5fbe4146]" name="teardown" is-config="true" duration-ms="12392" started-at="2020-06-28T10:00:30Z" finished-at="2020-06-28T10:00:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- Academy.validateNavigationBar -->
    </test> <!-- Validating navigation Bar -->
    <test name="Validating title in the Page" duration-ms="117889" started-at="2020-06-28T04:29:00Z" finished-at="2020-06-28T04:30:58Z">
      <class name="Academy.validateTitle">
        <test-method status="PASS" signature="initialize()[pri:0, instance:Academy.validateTitle@609cd4d8]" name="initialize" is-config="true" duration-ms="86013" started-at="2020-06-28T09:59:00Z" finished-at="2020-06-28T10:00:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method status="FAIL" signature="validateAppTitle()[pri:0, instance:Academy.validateTitle@609cd4d8]" name="validateAppTitle" duration-ms="2579" started-at="2020-06-28T10:00:26Z" finished-at="2020-06-28T10:00:29Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [FEATURED CO123URSES] but found [FEATURED COURSES]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [FEATURED CO123URSES] but found [FEATURED COURSES]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at Academy.validateTitle.validateAppTitle(validateTitle.java:43)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:37)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:368)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateAppTitle -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:Academy.validateTitle@609cd4d8]" name="teardown" is-config="true" duration-ms="9722" started-at="2020-06-28T10:00:48Z" finished-at="2020-06-28T10:00:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- Academy.validateTitle -->
    </test> <!-- Validating title in the Page -->
  </suite> <!-- Suite -->
</testng-results>
